rundeckapp/grails-app/domain/rundeck/Option.groovy

```diff
@@ -52,7 +54,7 @@ public class Option implements Comparable{
     Boolean required
     Boolean isDate
     String dateFormat
-    SortedSet<String> values
+    Set<String> values
```

Comment:
Does changing this affect existing db entries at all? is there a need to change it since we are moving to saving the input string anyway?